pkg/getty/root/.profile

```diff
@@ -0,0 +1 @@
+PS1="(getty ns) \w \$ "
```

Comment:
👍 I'll see about moving it to a more global location so it can be used for both sshd and getty.
it'd be very handy to have some runes in `$PS1` updates based on which namespace you are in.